static/js/admin.js

```diff
@@ -110,6 +137,15 @@ exports.setup_page = function () {
         error: failed_listing_streams
     });
 
+    // Populate filters table
+    channel.get({
+        url:      '/json/realm/filters',
+        timeout:  10*1000,
+        idempotent: true,
+        success: populate_filters,
+        error: failed_listing_filters
+    });
```

Comment:
Same comment here as with realm_emoji -- the data is already fetched in the `/` query so we don't need the extra GET call here.